Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,59 @@
+
+import unittest
+import os
+import subprocess
+
+
+class Testing(unittest.TestCase):
+    # Running the container with correctly set environment variables, all tasks from run_tests.py should pass
+    def test_postive(self):
+        exec_cmd = """
+        /run_tests.py
```

Comment:
I know @samvarankashyap introduced such behaviour but IMHO it is invalid case when env IMAGE_TO_TEST
is not specified. Using default image for testing is wrong.
People could do a typo in env variable IMAGE_TO_TEST and we would test default image which will pass all checks. and they can assume their image is good.

We have to fail with error for missing env variable IMAGE_TO_TEST

Reply:
I would recommend to explicitly pass good image here.
And write new test-case to fail with error (Failure due to what is probably  a system or an infrastructure issue)
for missing env variable `IMAGE_TO_TEST`

Reply:
Refactored, added.

Reply:
@jsztuka I still cannot see the change.